examples/experimental/i3s-17-and-debug/url-utils.js

```diff
@@ -0,0 +1,8 @@
+export function createTilesetUrl(parsedUrl) {
```

Comment:
Typescript and JSDoc please.

Reply:
Should we move this into the loader as well?

Reply:
I don't think we should move it to loader because it is just helper function for debugging app. Also I can do` .ts` files with `JSDoc` but I don't see any` .ts` and `JSDoc` files in examples folder. Is it really necessary here?